aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/util/ResourceModelHelper.java

```diff
@@ -27,6 +27,14 @@ public static boolean shouldUpdateAfterCreate(final ResourceModel model) {
                 );
     }
 
+    public static boolean isRestoreToPointInTime(final ResourceModel model) {
+//         Parameters to rely on are UseLatestRestorableTime and RestoreTime to tell if this is a RestoreToPointInTime
+//         But TargetDBInstanceIdentifier, SourceDBInstanceAutomatedBackupsArn and DbiResourceId are also unique identifying parameters of RestoreToPointInTime
+        return (model.getUseLatestRestorableTime() != null && model.getUseLatestRestorableTime() == true) || StringUtils.hasValue(model.getRestoreTime())  ||
```

Comment:
I think this can be simplified to `StringUtils.hasValue(model.getUseLatestRestorableTime()) || StringUtils.hasValue(model.getRestoreTime())`

Reply:
Agree. Don't go hard on these validations: RDS API will do the job

Reply:
There is BooleanUtils.isTrue() function with that functionality

Reply:
Addressed